templates/docker-container.yml

```diff
@@ -6,14 +6,35 @@
 trigger:
```

Comment:
Modify the comments in the first three lines of the file to just one line -
"# Build and push an image to Azure Container Registry"

As it stands, right now this template and the configuration panel that is shown next to it only allow for pushing to an existing ACR. So this comment will reflect the same